### DISCUSSION TITLE
Add achievements

### DISCUSSION THREAD

**Discussion by jamiemaertens** [ehmorris/lunar-lander#31] [Ideas]
This game could really do well with achievements such as crash scores, flips, landing scores, chasing around indicator icons, etc. I do replay it quite a bit without it but if I had more goals, I'd likely try to achieve them. :)

**Comment by ehmorris** [comment]
Thanks for the suggestion!

**Comment by ehmorris** [comment]
This exists now! You accrue bonus points as you do flips and reach speed and height milestones
